pkg/utils/utils.go

```diff
@@ -23,3 +23,20 @@ func GetMapValue(key string, dataMap map[string][]byte) (string, bool) {
 	}
 	return "", false
 }
+
+// Determine whether the label is attached to the runner or the reporter.
+func AddMapValue(key string, value string, dataMap map[string]string, overwrite bool) map[string]string {
+	if overwrite {
+		_, ok := dataMap[key]
+		if ok {
+			dataMap[key] = value
+			return dataMap
+		} else {
+			dataMap = map[string]string{}
+			dataMap[key] = value
+			return dataMap
+		}
+	} else {
+		return dataMap
+	}
```

Comment:
@S-mishina 

Regarding `overwrite`, it's supposed to be like this?

- If overwrite is true, add the value for the key to the map even when the key already exists in the map  
- If overwrite is false, NOT add the value for the key to the map when the key already exists in the map

Reply:
Thanks for your comment!
Corrected.
 Is this recognition correct?
https://github.com/st-tech/gatling-operator/pull/63/commits/6ea218acef437c9ec307e3f6f9d3431e78b95720

Reply:
@S-mishina 
Please read https://github.com/st-tech/gatling-operator/pull/63#discussion_r1011364561 at first